core/src/components/slider/slider.ts

```diff
@@ -383,7 +445,9 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 			return true;
 		}
 		const maxLabelWidth = maxLabelWidth$();
-		return !valuesPercent$().some((percent) => percent > 100 - maxLabelWidth - 1);
+		return rtl$()
+			? !valuesPercent$().some((percent) => 100 - percent < maxLabelWidth + 1)
+			: !valuesPercent$().some((percent) => percent > 100 - maxLabelWidth - 1);
 	});
 	// TODO define the intersection value
```

Comment:
just for curiosity, do we still need the TODO?

Reply:
yep, haven't actually tested with the huge numbers, will come with user testing